contracts/.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
```

Comment:
Is this file accidentally committed? To work, I think it needs to be in the root's `.github/` dir

Reply:
foundry adds this by default. let me try to move it